plugins/scaffolder-backend/src/scaffolder/stages/prepare/preparers.test.ts

```diff
@@ -61,7 +61,7 @@ describe('Preparers', () => {
 
     expect(() => preparers.get(mockTemplate)).toThrow(
       expect.objectContaining({
-        message: 'No preparer registered for type: "file"',
+        message: "No preparer registered for type: 'file'",
```

Comment:
I dunno, I'd say we use double quotes in regular human readable text because that's the norm

Reply:
I find `"` to be best for quoted text, and `'` for identifier type things. Overall `'` is by far more common in the project atm